scripts/screens/cat_screens.py

```diff
@@ -915,7 +915,7 @@ def generate_column2(self, the_cat):
 
         # STATUS
         if the_cat.outside and not the_cat.exiled and not the_cat.status in ['kittypet', 'loner', 'rogue',
-                                                                             'former clancat']:
+                                                                             'former Clancat']:
```

Comment:
Is 'former Clancat' a valid status?
I thought 'exiled' is used for ex-Clancats.